Apps/frontend/src/views/Logo.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
+  import logo from "../assets/sosci-logo.svg";
+</script>
+
+<div>
+  <img src={logo} alt="SOSCI" />
```

Comment:
For the alt-text perhaps rather: text SOCSCI in front of a sine curve, which is attenuated at the beginning and end.

Reply:
both fixed!